src/MLabsPAB/PreBalance.hs

```diff
@@ -61,11 +61,11 @@ preBalanceTxIO ::
 preBalanceTxIO pabConf ownPkh unbalancedTx =
   runEitherT $
     do
-      utxos <- lift $ CardanoCLI.utxosAt @w pabConf $ Ledger.pubKeyHashAddress ownPkh
+      utxos <- lift $ CardanoCLI.utxosAt @w pabConf $ Ledger.pubKeyHashAddress (Ledger.PaymentPubKeyHash ownPkh) Nothing
```

Comment:
If we set the staking address to nothing, will the utxosAt call work for addresses with staking creds?

Reply:
need to test it for real to make sure, but I'd assume that it does.
utxosAt is actually making a call to the chain-index, which only accepts the pkh (or script validator hash)
https://github.com/input-output-hk/plutus-apps/blob/main/plutus-chain-index-core/src/Plutus/ChainIndex/Api.hs#L90-L94
https://github.com/input-output-hk/plutus/blob/c8c5183f7facd967d48fe07b3b14465b8dd48fe7/plutus-ledger-api/src/Plutus/V1/Ledger/Credential.hs#L53-L57

Reply:
Oh neat! Thats good to know